src/components/collapsible/Collapsible.js

```diff
@@ -119,9 +141,15 @@ class Collapsible extends React.Component<Props, State> {
                     </PlainButton>
                     {isOpen && headerActionItems}
                 </div>
-                <AnimateHeight duration={animationDuration} height={isOpen ? 'auto' : 0}>
-                    <div className="collapsible-card-content">{children}</div>
-                </AnimateHeight>
+                <div
+                    ref={this.ref}
+                    style={{
```

Comment:
Ideally, the non-dynamic portions of these styles should live as part of a class in the stylesheet rather than as inline styles.

Reply:
+2

Just need to have transition-duration here